### PR TITLE
Revert "DirInserter always inserts root (#6277)"

### PR DESCRIPTION
**Pull request by brendoncarroll**
This reverts commit b58ffbdcb66dd946b87dd5abd5f9173501e296e0.

### CODE REVIEW

File: src/client/pfs_file.go [hunk 1]
```diff
@@ -701,9 +701,6 @@ func (c APIClient) DiffFileAll(newCommit *pfs.Commit, newPath string, oldCommit
 
 // WalkFile walks the files under path.
 func (c APIClient) WalkFile(commit *pfs.Commit, path string, cb func(*pfs.FileInfo) error) (retErr error) {
-	defer func() {
-		retErr = grpcutil.ScrubGRPC(retErr)
-	}()
```
Comment (Tryneus) [inline src/client/pfs_file.go:706]: Why was this removed?  I think we still want to scrub the error here, unless we're going to do a pass across all the client code and make this consistent.

**Review by Tryneus — APPROVED** [review]
One nit, otherwise LGTM

File: src/client/pfs_file.go (hunk 1 above)
Comment (brendoncarroll) [reply src/client/pfs_file.go:706]: This was removed so that the gRPC status code can be inspected.  The test now checks for that.  I don't think we need to move all the error handling over right now.
